mypy/errors.py

```diff
@@ -1095,7 +1095,12 @@ def remove_duplicates(self, errors: List[ErrorTuple]) -> List[ErrorTuple]:
                         conflicts_notes = True
                     j -= 1
                 j = i - 1
-                while j >= 0 and errors[j][0] == errors[i][0] and errors[j][1] == errors[i][1]:
+                while (
+                    j >= 0
+                    and errors[j][0] == errors[i][0]
+                    and errors[j][1] == errors[i][1]
+                    and errors[j][2] == errors[i][2]
+                ):
```

Comment:
this is confusing, can you give the variables more descriptive names

Reply:
take it to upstream general, Isn't it industry standard to name loop variables i and j?